lib/components/DynamicFonts/index.ts

```diff
@@ -173,4 +173,23 @@ export default class DynamicFonts {
   public async isFontDownloaded(fontName: string, fontExtension: FontExtension): Promise<boolean> {
     return await this.fontDownloader.isFontDownloaded(fontName, fontExtension);
   }
+
+  public async isFontFamilyDownloaded(rootUri: string,
```

Comment:
It sounds like you named the method with the purpose you need it for while technically this method checks if multiple fonts are downloaded, right?

Reply:
It checks if the whole font family is downloaded, yes; all the methods in this API are this way.
We can try to think of a different API if you like, but I'm merging for now.